### DISCUSSION TITLE
Does 3.2.2 apply only to form elements, because only for them a value can be changed

### DISCUSSION THREAD

**Discussion by JAWS-test** [w3c/wcag#2647] [Q&A] [answered]
3.2.2:

> Changing the setting of any [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components) does not automatically cause a [change of context](https://www.w3.org/TR/WCAG21/#dfn-change-of-context) unless the user has been advised of the behavior before using the component.

Is my assumption correct that 3.2.2 does apply only to form elements and e.g. not apply to menus, because menus have no value (except maybe menuitemradio and menuitemcheckbox). I.e. navigating through a menu does not count as "Changing the setting"

And what about elements that can be both: Form element or navigation element. For example, a tree : it can be used to select a value or for navigation, like a link list. Then, depending on the context, do we have to decide whether 3.2.2 is applicable or not?

**Comment by patrickhlauke** [comment, marked as answer]
yes. in a menu, you actively activate an menu item, you're not "changing the value" of the menu.

menuitemradio and menuitemcheckbox though bring up an interesting point. i'd say that *those* (together with things like an `aria-pressed` toggle button) would count / fall under 3.2.2 ... if toggling/checking them does result in a change of context (other than, in the case of the menu ones, closing the menu again and setting focus back to the menu trigger), that *would* fall under 3.2.2
